### PR TITLE
Add Bezier unit tests

### PR DESCRIPTION
**Pull request by hannahli2010**
Add tests for the Bezier functions that do not have any yet.

### CODE REVIEW

File: bezier-rs/lib/src/lib.rs [hunk 1]
```diff
@@ -767,17 +781,320 @@ mod tests {
 		assert!(compare_points(bezier3.evaluate(0.), p2));
 	}
 
+	#[test]
+	fn evaluate() {
```
Comment (Androxium) [inline bezier-rs/lib/src/lib.rs:785]: did we want to prefix our tests with `test_`?

**Comment by Keavon** [comment]
@hannahli2010 it looks like a number of these tests only try the endpoints (`t = 0` and `t = 1`). Testing the middle (`t = 0.5`) would be good for those. Maybe you can find an online Bezier calculator, or hook it up to Bezier.js, to double check some of those values to avoid dogfeeding your tests with your implementation results.

File: bezier-rs/lib/src/lib.rs [hunk 2]
```diff
@@ -1228,16 +1223,362 @@ mod tests {
...
+		assert!(rotated_bezier.abs_diff_eq(&expected_bezier, MAX_ABSOLUTE_DIFFERENCE));
+	}
+
+	#[test]
+	fn test_translate() {
+		let bezier = Bezier::from_cubic_coordinates(30., 30., 60., 140., 150., 30., 160., 160.);
+		let translated_bezier = bezier.translate(DVec2::new(10., -10.));
+		let expected_bezier = Bezier::from_cubic_coordinates(40., 20., 70., 130., 160., 20., 170., 150.);
+		assert!(translated_bezier.abs_diff_eq(&expected_bezier, MAX_ABSOLUTE_DIFFERENCE));
 	}
```
Comment (Keavon) [inline bezier-rs/lib/src/lib.rs:1582]: Please test rotate and translate with linear, cubic, and quadratic.

**Comment by Keavon** [comment]
I'm going ahead and merging this now to avoid keeping the branch open too long and getting stale. For the remaining code review comments I left, please open another PR when you guys have the time. Thanks!

**Pull request merged**